Dockerfile

```diff
@@ -167,6 +167,11 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update \
         libxml-simple-perl \
         libnet-sip-perl \
         libxml-stream-perl \
+        libhtml-strip-perl \
```

Comment:
Kindly ensure proper sorting of the added lines in order to keep the clear structure and better overview.